ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -204,9 +190,9 @@ class JITModBuilder {
             // have no IR, and thus we cannot map blocks for.
             ExpectUnmappable = true;
             // Peek at the instruction we will need to resume at.
-            auto PeekIter = I;
-            PeekIter++;
-            ExtCallResume = &*PeekIter;
+            // auto PeekIter = I;
```

Comment:
Commented code.

Reply:
Seems this code went away in a later commit. Shall I "go back in time and remove it there too"?

Reply:
I think if you edit the rebase at the first commit and remove the code, it will magically disappear from the subsequent commit entirely. It might be worth trying first quickly! If it doesn't work easily, this change isn't strictly necessary.